pkgs/by-name/ot/otree/package.nix

```
@@ -0,0 +1,41 @@
+{ lib
+, fetchFromGitHub
+, rustPlatform
+, stdenv
+, git
+}:
+
+rustPlatform.buildRustPackage rec {
+  pname = "otree";
+  version = "0.1.0";
+
+  src = fetchFromGitHub {
+    owner = "fioncat";
+    repo = "otree";
+    rev = "v${version}";
+    hash = "sha256-ukKlypGY0+xsci/loAnf0Sz5K+82gVzkb+KnZ139cIA=";
+    leaveDotGit = true; # needed by build.rs
+  };
+
+  nativeBuildInputs = [ git ];
+
+  cargoLock = {
+    lockFile = src + "/Cargo.lock";
+    outputHashes = {
+      "tui-tree-widget-0.19.0" = "sha256-icHGxbqRqEo3XXst3SGJ5q6G3fg6In0Lk6vXdJtJTvc=";
+    };
+  };
+
+  doCheck = false; # there's no tests
```

Minor correction of grammar
```suggestion
  doCheck = false; # there are no tests
```